### PR TITLE
String Quartet No. 8, Dimitri Shostakovich

### PR DESCRIPTION
**Pull request by nitrogenfella**
(String Quartet No. 8)

(I. Largo)
This code is SHIT. Your server is SHIT. Your admins are fucking SHIT. Some of them are ok, yeah, but then some of them are just power hungry, or just plain fucking boring weird people, Delta types like a fucking neanderthal and hides behind some false visage of being ESL. Like, come the fuck on, dude. 

Your code is so fucking abysmally fucking lazy. I work, I work a crushing 6-5 job, I work abysmal fucking conditions that in violation of basic union laws, and even I, EVEN FUCKING I have time to do mapping work that is BETTER than NINETY PERCENT OF THE SHIT uploaded into these SHITTY CODEBASES. These absolute DOGSHIT RANK HORIRBLE codebases. You have all of you, all of you NEET motherfuckers, none of you even have any obligations or real duties beyond fucking Tadano who has a family to care about, the rest of you are FUCKING LIFELESS! Like ohh yeah I have a job but You don't spend ANY OF YOUR FREE TIME ON THIS SHIT AND IT SHOWS!!! It's so FUCKING LAZY!! A YEAR on a fucking rebase, and for WHAT? It's not EASIER to code with, the CODE ISN'T CLEANER. Oh yeah, I guess it RUNS A LITTLE BETTER? But that's YOUR FUCKING FAULT FOR USING COYOTE BAYOU WHEN WE SHOULD HAVE JUST USED FUCKING ATOM BOMB FROM THE START!! AND DON'T HIT ME WITH THAT, "WAAAAHH SO AND SO DELETED THE REPO-" EVERYONE STILL HAS IT FORKED, MAN! I STILL GOT IT FORKED! EVERYONE DOES! And then, and then you take, ALL THE GOOD work gone into Nukapop, that absolute PIPEDREAM of a server and you SQUANDER it. You let your ENDLESS HOARDS of RETARDS squander it until it's ALL GONE. Your code, your repo, your project, it is so abysmally fucking lazy. Fallout 13 is a tower, it is a tower of corpses, corpses of different contributors, some died because they're lazy fucking cunts, some died because of weird fucking pedo drama, some died because they became utterly frustrated with the community and other developers. 
And it's ALL SHIT. ENDLESS MOUNTAINS OF DOGSHIT SPRITES, ENDLESS MOUNTAINS OF CONCEPTS ADDED BY UTTER FUCKING SECONDARIES WHO HAVE NEVER EVEN PLAAAYED A FUCKING FALLOUT GAME! So much fucking RANK, EVIL, DISGUSTING SLOP INSIDE OF THIS SHIT- and then you, you people, you do some good! You get rid of the ERP slop, wow, so brave! So brave! So brave of you to not let people typeesx and be fucking deegenerates! But you DO ABOUT JACKSHIT FUCKING ELSEEEE! IF YOU HAD A FUCKING COMPETITION BETWEEN A TURTLE AND A FALLOUT 13 DEVELOPER AS TO WHO DOES LESS WITH THEIR TIME THE FUCKING DEVELOPER WOULD WIN!! THE TURTLE WOULD LOOK LIKE THE BUSIEST MAN ON FUCKING PLANET EARTH!! YOU PEOPLE DO NOTHING!! YOU ARBITRARILY CHANGE TINY NUMBERS, YOU REMOVE THINGS WE LIKE, YOU OCCASIONALLY ADD THE NEAT SMALL FEATURE, BUT ULTIMATELY YOU ADD FUCKING NOTHING!! THE FALLOUT 13 OF NOW IS ALMOST ENTIRELY SIMILAR TO FUCKING OUTCASTS!! THE ORIGINAL FUCKING OUTCASTS SERVER!! NOTHING HAS CHANGED!! YEARS OF FUCKING RANCID FUCKING SLOP, YEARS OF LAZY INCONSISTENT DEVELOPMENT, YEARS OF FUCKING CHILDREN AND SECONDARIES WHO DON'T KNOW THE SLIGHTEST THING ABOUT THE FUCKING GAME WORLD THEY ARE SUPPOSED TO BE ROLEPLAYING IN. 

And then the things you do that, they are so abysmally dogshit. So abysmally dogshit. From the very START of considering a SPECIAL system, EVERYONE, and I mean EVERYONE. ADOLF HITLER, JOSEPH STALIN, THEY'D BOTH AGREE AND SAY, YEAH, WE AGREE THAT HAVING A SPECIAL SYSTEM WHERE PEOPLE MAKE MINMAX CHARACTERS is FUCKING ASS!! Even MORE SO when you want a FUCKING ROLEPLAY GAME!! Even MORE SO when the ENTIRE FUCKING GAME is ABSOLUTELY FULL OF CONSUMABLE DRUG ITEMS THAT TURN YOU INTO FUCKING SUPERMAN, NO, NO, NOT SUPERMAN, FUCKING OMNIMAN? THANOS WITH THE FIVE STONES? GOD HIMSELF? I"M OUT HERE IN FULL POWER ARMOR WHILE A CRACKED OUT, MINMAXXED, FEMSTATIC, KOREAN LOOKING ANIME CHARACTER KILLS ME IN THREE HITS WITH A FUCKING SHITTY SLEDGEHAMMER THEY CRAFTED TEN MINUTES INTO THE ROOUND!! WHAT THE FUCK WERE THEY THINKING? I mean,REALLY? What WERE you thinking? And then don't even get me started on Train's stupid fucking ass, adding this whole stupid chicken system and for WHAT?

(II. Allegro molto)
 NO ONE uses it! IT'S NOT REMOTELY FUCKING FALLOUTY! WHERE'S THE SCENE IN FALLOUT 2 WHERE YOU RAISE A FUCKING ELECTRIC TYPE CHICKEN YOU FUCK? IS THIS FUCKING POKEMON? OOOOH I GUESS THIS IS THE POKEMON SERVER! I'LL TRADE YOU MY DICK IN YOUR FACE MOTHERFUCKER. ALL OF YOU HAVE NO IDEA WHAT THE FUCK A FALLOUT EVEN IS OR HOW THE FUCK IT TRANSLATES INTO A MULTIPLAYER GAME! IT'S ALL A WASTED FUCKING OPPORTUNITY! SERIOUSLY, EXPLAIN TO ME, HOW THE FUCK DO GENETICALLY BRED POKEMON ASS POULTRY CONTRIBUTE TO A FALLOUT EXPERIENCE? AND THEN YOU'LL SAY, "BUT OH, WE HAVE DONE SO MUCH ! WE HAVE DONE SO MUCH" ALL YOU'VE FUCKING DONE IS TAKE A BROKEN ASS CODEBASE, SHAMBLE AROUND WITH IT FOR A YEAR, MAKE IT WORSE IN EVERY FUCKING WAY UNTIL IT BECAME *SOOOO FUCKING UNBEARABLY DOGSHIT* THAT YOU HAD TO JUMP TO ANOTHER CODEBASE BECAUSE IT KEPT CRASHING EVERY TEN FUCKING MINUTES!! AND THIS CODEBASE IS EVEN WORSE AND YOU'RE MAKING IT EVEN MORE SHIT! I"D RATHER EAT THE ROTTEN ASSHOLE OF A ROADKILL SKUNK THAN SERIOUSLY CALL THIS ANYTHING OTHER THAN ABYSMAL DOGSHIT! 
 And of course, You'll READ THROUGH THIS and say, OH, WHAT A MADMAN! YELLING, SO MAD! I'M yelling because I WANT A GOOD FUCKING FALLOUT GAME and you MOTHERFUCKERS DO ABSOLUTELY NOTHING WITH YOUR BUREACRATIC CHANNELS! NOTHING EVER HAPPENS!! YOU IGNORE EVERYTHING PEOPLE SAY, OR YOU FIND A WAY TO CLAIM THAT THEY'RE WRONG, OR YOU SHOVE IT INTO AN ENDLESS FUCKING BACKBURNER OF SHIT THAT IS NEVER FINISHED!!! YOU MAKE NO ATTEMPT TO SPARK ACTUAL DEVELOPMENT OR A WANT TO ENGAGE WITH FUCKING DEVELOPMENT!! IT TAKES YOU MONTHS TO FIX THE SMALLEST FUCKING BALANCE ISSUES ALL THE WHILE YOU COMPLETELY IGNORE THE COMMUNITY WITH FEEDBACK ABOUT THE SPECIAL SHIT! FIND ME ONE, FIND ME ONE FUCKING PERSON WHO GENUINELY AGREES WITH YOU THAT THE CURRENT SPECIAL SYSTEM IS GOOD? NONE! FIND ME ONE PERSON WHO THINKS BALTIMORE IS EVEN REMOTELY GOOD COMPARED TO YUMA? NONE! AND I KNOW THAT DELTA"S CLOVEN IN HEAD ASS WILL REPLY IN SOME ANGRY DISCONNECTED ENGLISH ABOUT HOW OHHH NOO IVE ONLY BEEN WORKING ON IT FOR A SHORT TIME, A SHORT TIME? IT'S BEEN SIX FUCKING MONTHS!!! I COULD MAKE THE FUCKING MONA LISA OF MAPPING IN THAT TIME YOU MOTHERFUCKER !! YOU'RE LITERALLY LIKE A WEIRD NEET, AREN'T YOU? YOU HAVE ALL THE TIME IN THE WORLD! BUT SO MANY OBJECTIVE THINGS ABOUT THE MAP ARE JUST SHIT! IT'S TOO CROWDED! ALL THE FACTION BASES LOOK LIKE ASS! THERE'S NO RECOGNIZABLE LOCATIONS! YOU HAVE A BILLION ZILLION MILLION ENEMIES IN EVERY FUCKING BUILDING! Oh, oh, good sir, I would like to loot this soup shop, SORRY JACKASS! YOU GET FIFTY GHOULS SPAWNING IN ON YOU AND PROBABLY TANKING THE FUCKING PERFORMANCE!! AND then Don't even get me started how the ENTIRE. THE ENTIRE. THE ENTIREEEEEEEE FUCKING COMMMUNITY AGREED UP AND DOWN THAT THE ENCLAVE WAS ASS TO HAVE ON YUMA, AND SO MANY MORE AGREED THAT THE BROTHERHOOD NEEDED TO bE REDUCED ON YUMA, TOO, WE LITERALLY GOT BALTIMORE SOLD TO US ON THE IDEA THAT THE ENCLAVE CUCKS AND BROTHERHOOD CUCKS WOULD BE CONFINED TO THAT SHITTY FUCKING GHETTO ASS MAP WITH COBBLED TOGETHER ASSETS THAT ALL LOOK LIKE ASS OTHER THAN THE ROAD SPRITE THAT WAS PROBABLY STOLEN FROM SOMEWHERE 
 
 (III. Allegretto) 
 It's so unbearably ass. I can't fathom how you fucking people have let any of it get this bad. This code has been in the absolute unredeemable shitter since Wasteland and all of you have been making it worse. Purposeful backdoors in the code. Stealthbuffs to your favorite little faction. Items and shit that blatantly break the fucking narrative consistency of fallout, a braindead fucking community who hasn't even played the fucking games. You SICKEN me. What fucking fallout fans are you when you've made a WORSE game than Fallout 76? WHO are you to have done that? I can boot up that shitpile crashgrab known as Fallout 76 and have loads of more fun than this fucking dogshit, this dogshit that has had YEARS of effort put into it. YEARS!!! YEARS!!! AND FOR FU CKING NOTHING!! !!!!FOR A DOGSHIT CODEBASE WITH A DOGSHIT BALANCE WITH A DOGSHIT ART STYLE!! And when SOMEONE ORIGINAL like Mojave SUN appears? You all SCORN IT YOU ALL DO EVERYTHING IN YOUR POWER TO SHIT TALK IT, TO SMEAR IT, TO FUCK WITH IT, TO MAKE BAD REDDIT POSTS ABOUT IT. Yeah I'm SORRY that MS PEOPLE ARE SNARKY about their SERVER being better than YOUR SHITPILE!! BECAUSE IT FUCKING IS!!! MOJAVE SUN IS LIKE A HALF EDIBLE SHITTY BURGER AT A MCDONALDS WHILE YOUR CODE IS A FUCKING HEAPING PILE OF RANCID RANK FUCKING HORSE DUNG ON A ROTTEN HALF SKELETAL WHALE THAT WAS FEASTING UPON ROTTEN MEXICAN FOOD IT'S ENTIRE FUCKING LIFE!!! It's so FUCKING LAZY! EVERY FUCKING INCH of this CODE is just full of LAZINESS! Absolute and UTTER fucking LAZINESS. Look at the mapping, look at all these illogical fucking buildings that had to have been mapped by children or the mentally handicap, all of these illogical fucking places that have no reason to exist, no cohesion to exist, I'm BARELY EVEN AN EXPERIENCED MAPPER AND I CAN SHIT OUT SOLID FUCKING GOLD IN COMPARISON TO THIS COAL! THIS VANTABLACK FUCKING HYPERDARK COAL!
 
 (IV. Largo)
 Fallout 13 is an abysmally lazy joke of an open source project. What began as something with a hint of promise was utterly ruined by drama, children, idiotic morons, and lazy dumbfucks who seize power and trample over anyone beneath them. You're not all bad people, most of you aren't, but a good amount of you are fucking lazy. You can shit talk to me all you want, you can insult me, you can call me CRAZY, you can call me COOCOO FOR COCOA PUFFS but you can't call me FUCKING LAZY. almost EVERY fallout server I've had my hand in trying to give feedback, or bug test, or report bugs, or FUCKING MAP! I have TRIED to CONTRIBUTE, I have ASKED to be taught how to code before and NOTHING comes of it because you're all fucking LAZY. LAZY or you're people who are too busy to be holding power over something like this. This server, this Big Iron shit, it's a fucking *cancer*. If it were to die, give it a month, and a new server would take it's place. A new server that, at the very least, would offer a change of some things. Instead of this endless fucking stagnation. Instead of this endless meaningless dogshit, this unplayable abysmal dogshit. And it's not just the developers fault, it's the fucking community. Genuinely fuck at least half of you for either being brainrotten children who have never played a fallout game or leaky-brained fragbabies that only ever cared about minmaxxing your build in a fucking prehistoric spaceman fallout game only to make everyone else's lives worse. I wanted, what a lot of us want, is a fucking fallout immersive sim, what we want is a fucking RP experience, and every server ever has shat on that, has kicked it, or has done it in the most utterly wrong way. Crashpoint came the closest, and Mojave Sun tried their own thing, but both ultimately fucking died. And I'm DONE! Let this be my EPITAPH at ever bothering with these fucking fallout servers again. If you read this, Pesta, I'm not going to map for your stupid fucking reno server or whatever gay project you have now that will never materialize into anything. Fallout 13 is fucking dead and has been since Wasteland opened it's stupid fucking doors, we've all just been playing with the corpse, parading it around, putting on makeup, act like we're healing the corpse by sawwing off a finger and sewwing another one on. It's all a big heap of fucking rubbish. All a giant fucking nothingburger. I'm going to go spend the rest of my weekend doing something productive, doing something happy, and doing something that isn't thinking about your heaping pile of dogshit that you call Fallout 13. Fuck you, Desert Rose pedophiles. Fuck you, everyone who enabled them. Fuck you, furry weirdos for splintering the playerbase, fuck you, lazy ass developers across all these servers, fuck you, Bob, for being an irratic pedophile who quits every project he devlops for (the bob i'm talking about will know who he is). Fuck you, Bobnewman, for being a usual discord janny. Fuck you, Train, for being a lazy motherfucker who implements constant dogshit vantablack coal. Fuck you, Hekzder, for having a fucking period and abandoning MS in the dust. Fuck you, Scar, for being an irratic alcoholic assfuck who was so insane that you got kicked out of your own friend group. Fuck you to every motherfucker who never even so much as tried to help with their own vision of a Fallout 13. Fuck you, Ghostler, you fucking snowflake dipshit with your anime ass characters, constantly acting like you're hot shit, the most robust to ever exist. Fuck you, Pug, you're all around a shit person and have literally told me to kill myself multiple times just because I'm homosexual. You also play these constant snowflake anime lesbians and I'm so fucking tired of it. Fuck you to all of the ESL children who made it impossible to ever truly roleplay or enjoy this game when I wanted to. Fuck you to all of the simpletons who willl never do a single fucking thing to further any of the things they love, including Fallout 13. Fuck you to Technobug for being a lazy cunt more obsessed with fucking his e-girlfriend on rougetown. Fuck you to Original for throwing in the towel and acting like it was MYYY FAULT that I didn't know what the fuck to map on your maps that were already completely finished and had no space for improvement. Fuck you, Rebel0 or whatever your name is, an absolute tumblrite power monging dipshit. Fuck you, Raptor, you're way in over your head and should have never bothered managing a server. Fuck you, fuck you, fuck you, fuck you. I'm going down to the carniceria for some beer and BBQ. Good luck with your miserably dogshit server. Your miserably dogshit playerbase. Your miserably dosghit code. Your miserably dogshit developers. Have a nice day. Goodbye.

### CODE REVIEW

**Comment by BobNewman-max** [comment]
That's a long wall of text, can you tell me what it does in a TL;DR?

**Comment by DeltaDav** [comment]
Honnestly, some points of that wall of text are true, but other need a lil more thinking. You need to get some of your fact checked. 
Look, its true that I write in a bad english, and that I am often not here and unresponsive to suggestions in those last 3 months. I had other things going. That I am sorry about, but thats how life works. 

Baltimore is the first full map I have ever done. Thats no secret. And I had to make it fast. Even I think that Yuma is better. But I did it to contribute to F13 and the server. To offer something new. Lot of folks find it okay, others don't, it was expected. Will keep working on it. Baltimore might get replaced with graf's superior east coast map when it is released, and I fully support that decision. But at least I did my best to contribute to a project I trust in. 

Its okay to be frustrated. Have a nice day too. 
https://www.youtube.com/watch?v=-rsEs4HWXeY

**Comment by Loldaddy5676** [comment]
> That's a long wall of text, can you tell me what it does in a TL;DR?

TL;DR- This guy apparently was so angry for the past few months that rather then just ignoring a server where he hates the admin team and dev team and how they work........

**Comment by Aurrain** [comment]
He's mad at Mojave Sun, not us really.
We just happen to be the only place remaining, currently.

Anyway, see you around.

**Comment by CaputCapital** [comment]
I'm only here because somebody linked me this and wanted me to see it. This is just a schizopost/shitpost that took too much time to put together. As to why and what the fuck are we trying to analyze this schizo/shit for, I don't know because from what it looks like this edits part of the Pahrump map.

**Comment by Havingfun80** [comment]
I think it's time for u to look for other games
